### PR TITLE
Add KAT checks

### PR DESCRIPTION
**Pull request by smashra**
Ready to merge to master.

### CODE REVIEW

File: .travis/kat-check.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+###
+# Checks that all KATs pass.
+###
+
+PRINT_GREEN="tput setaf 2"
```
Comment (drucker-nir) [inline .travis/kat-check.sh]: I suggest to import .travis/defs.sh instead of duplicating code.

Comment (smashra) [reply .travis/kat-check.sh]: Done.

**Comment by smashra** [comment]
Ready to merge.

**Comment by smashra** [comment]
Windows build fixed.

**Pull request merged**